interface/src/app/analyze/analysis/analysisDetail.tpl.html

```diff
@@ -39,7 +39,7 @@
         <th>Sample Name</th>
         <th ng-repeat="at in analysis.exampleCols">{{
           at.typename
-          }}</th>
+        }}</th>
```

Comment:
Hmmm, just curious - why did you decide to format it like this instead of:
```
        <th ng-repeat="at in analysis.exampleCols">
          {{at.typename}}
        </th>
```
It is a minor quibble, but I wonder if it would help with readability, as interpolation markup in angular is usually shown as `{{variable}}`.

Reply:
I also prefer `{{...}}`.

Reply:
I am doing it this way to keep the resulting HTML markup as clean as possible. Let's say that `at.typename` resolves to "genotype." Then the way you suggest (which, I agree, makes the code more readable) will result in the following HTML:

```html
<th ng-repeat="...">
  genotype
<th>
```

This puts space between the markup tag and the contents, so the browser effectively reads this as:

```html
<th> genotype </th>
```

What I have written will keep the spaces out of the generated html to produce:

```html
<th>genotype</th>
```

This might not make much of a difference in the current layout, but consider what happens when you have:

```html
<a href="blahblahblah">
  {{something that resolves to "genotype"}}
</a>
```

The resulting HTML will be:

```html
<a href="blahblahblah"> genotype </a>
```

and this produces the dreaded underline styling that leaks around the text with the anchor. Something like

`_genotype_`

Performing CSS styling tweaks will be *much* easier with clean HTML.

Reply:
This is a good point. (Actually in `<th>...</th>` the leading and trailing spaces between the two tags will be ignored by default.) But I have to confess that it still looks weird to me.

Reply:
Ah gotcha, ok, that makes sense to me as well.

Reply:
This is true. The *default* styling for a `<th>` tag will not reveal the space between the tag and its contents. BUT: what if we wanted the column headers to appear as links because they should be clickable? This is not an uncommon practice... many tables will allow a user to sort by column by clicking the header of that column. This feature can be added to a plain table like this without rewriting the HTML through CSS and JavaScript, but it would only look right if the HTML were "clean."